src/services/quiz.service.ts

```diff
@@ -39,8 +38,14 @@ export async function getQuizByQuizBankId(id: string) {
     })
 }
 
-export async function copyQuizBankToClassroom(token: string, quizbankId: string, classroomId: string) {
-  const url = getAPIServerURL(`/api/classrooms/copyquizbank/${quizbankId}/${classroomId}`)
+export async function copyQuizBankToClassroom(
```

Comment:
Service is not so much meaning here. We agree on calling a api from server with mark use server is "action".
Service is for communication with server
While action is posting from client to nextjs server, then bridge to dotnet server. 
I dont want to diverse the package into server. stick with our got